pandas/tests/extension/test_arrow.py

```diff
@@ -225,6 +225,24 @@ def data_for_grouping(dtype):
     return pd.array([B, B, None, None, A, A, B, C], dtype=dtype)
 
 
+def expected_inferred_result_dtype(dtype):
```

Comment:
This depends on the UDF as well as the input dtype - so it could be different for different tests. I think this should be inlined in the one test it's used in instead.

Reply:
resolved